.github/workflows/test.yml

```diff
@@ -37,15 +37,6 @@ jobs:
 
       - run: pip install pipenv
 
-      # Copy-paste from
-      # https://github.com/actions/cache/blob/main/examples.md#python---pipenv
-      - uses: actions/cache@v2
-        with:
-          path: ~/.local/share/virtualenvs
-          key: ${{ runner.os }}-pipenv-${{ hashFiles('Pipfile.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-pipenv-
```

Comment:
Restoring the cache appears to be broken today. If this is present, the `python` executable disappears from the rest of the workflow, which is... a bit of a problem.

Reply:
Fixed in #233